ExperimentalAverage.c

```diff
@@ -60,9 +60,9 @@ int main (int argc, char *argv[]) {
 			 * The file is looped over using the while (x!=EOF) condition, and the first step we perform is check if
 			 * the next char is the end of the file.
 			 */
-			int x;
 
-			while (x != EOF) { // This is the loop for the
+			while (1) { // This is the loop for the
+                int x;
```

Comment:
From what I remember, I had a check here because on [line 67](https://github.com/ShivanKaul/CSVReader/blob/master/ExperimentalAverage.c#L67), I wanted to make sure I wasn't calling fgetc() after EOF was already reached [but it seems that once EOF is reached, the command would keep returning EOF so we're safe](https://stackoverflow.com/questions/4753869/is-getc-defined-after-it-has-returned-eof). This is cleaner. 

Why do you think x should be reinitialized in every iteration, though?